plugins/sdm/accessbot.py

```diff
@@ -193,6 +201,36 @@ def show_roles(self, message, match):
             return
         yield from self.get_show_roles_helper().execute(message)
 
+    @re_botcmd(pattern=r"whoami", flags=re.IGNORECASE, prefixed=False, name="accessbot-whoami")
+    def whoami(self, message, match):
+        """A simple command echoing the details of your identifier. Useful to debug identity problems."""
+        frm = message.frm
+
+        resp = ""
+        if self.bot_config.GROUPCHAT_NICK_PREFIXED:
+            resp += "\n\n"
+
+        resp += f"| key       | value\n"
+        resp += f"| --------  | --------\n"
+        resp += f"| person    | `{frm.person}`\n"
+        resp += f"| nick      | `{frm.nick}`\n"
+        resp += f"| fullname  | `{frm.fullname}`\n"
+        resp += f"| client    | `{frm.client}`\n"
+        resp += f"| email     | `{frm.email}`\n"
+
+        email_slack_field = self.config['EMAIL_SLACK_FIELD']
```

Comment:
Consider email subaddressing, override email and other one I can forget about... Reuse the function that we already with all this logic.